dev-libs/pigpio/pigpio-66.ebuild

```diff
@@ -27,7 +27,7 @@ src_prepare() {
 
 src_compile() {
 	emake
-	if use python ; then
+	if use python && distutils-r1_src_compile ; then
```

Comment:
no, it should just be `use python && distutils-r1_src_install`, `if` construction is not needed here anymore at all